plugins/mimecast/komand_mimecast/connection/connection.py

```diff
@@ -43,9 +43,24 @@ def test_task(self):
         self.logger.info("Running a connection test to Mimecast")
         try:
             _, _, _ = self.client.get_siem_logs("")
-            self.logger.info("The connection test to Mimecast was successful")
-            return {"success": True}
+            message = "The connection test to Mimecast was successful"
+            self.logger.info(message)
+            return {"success": True}, message
         except ApiClientException as error:
-            self.logger.info("The connection test to Mimecast has failed")
+
+            return_message = ""
+
+            failed_message = "The connection test to Mimecast has failed"
+            self.logger.info(failed_message)
+            return_message += f"{failed_message}\n"
+
+            cause_message = f"This failure was caused by: {error.cause}"
+            self.logger.info(cause_message)
+            return_message += f"{cause_message}\n"
+
+            assistance_message = f"This failure is fixed by: {error.assistance}"
```

Comment:
I think most of the assistances don't fit in with this prefix "This failure is fixed by:". Could just append the assistance to the end? At the minute in the PR reads as "This failed is fixed by: Please ensure that correct APP ID is supplied"

Reply:
this has been updated to remove the extra wording so it should now read better